src/test/javascript/spec/service/exam-participation.service.spec.ts

```diff
@@ -65,6 +65,16 @@ describe('Exam Participation Service', () => {
         const req = httpMock.expectOne({ method: 'GET' });
         req.flush(returnedFromService);
     });
+    it('should load exerciseIds of a student exam', async () => {
```

Comment:
avoid using `async ` in tests, you can use `fakeAsync `instead of it... I think we also need to `tick()` after `req.flush`

Reply:
Good catch. 👍
I just oriented myself on the other tests in the same file, but yeah they are also wrong ^^
I fixed mine.

Reply:
alright, then we need to take care of other tests in that file. it can be fixed in the context of another PR.